docs/modules/ROOT/pages/features.adoc

```diff
@@ -0,0 +1,134 @@
+= Features
```

Comment:
I did a few screenshots for the Chrome WebStore page:

<img src="https://lh3.googleusercontent.com/am_TCcxS0oJiRQ909Tg8ajwWFoRGJPjPwH5S4LfQr4tK_mymFmNi9aa2ThH_PN40Bk0Q0hRf4UHTL6ExRLO_zuTjDw=w640-h400-e365-rj-sc0x00ffffff"/>

We might want to use them. I can also take another of screenshots.

Reply:
Reusing the screen captures from the others makes for a consistent experience.

I did like the animated gif on the vs-code extension:  https://github.com/asciidoctor/asciidoctor-vscode#readme

Reply:
Let's use what you've taken before, for consistency, or I'm also happy to postpone the decision until the dust settles :-)